.gitignore

```diff
@@ -222,7 +222,7 @@ docs/_build
 
 ### Project ###
 /data/
-/texar_download/
+texar_download/
```

Comment:
Since we added one more folder layer (`torch`) in `texar`, `texar_download` is currently in `texar` folder due to function `default_download_dir()` in `pretrained_base.py`. I wll update `default_download_dir()` accordingly.

Reply:
I see. Yes, that would be great.